frontend/src/App.js

```diff
@@ -16,11 +16,25 @@ class App extends Component {
 
   render() {
     return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
-        </div>
+      <div className="Home">
```

Comment:
v nit: can we make it consistent whether classnames begin with lower case or upper case? (ex. `home` and `home-content`)